devops/playbook.yml

```diff
@@ -63,10 +63,17 @@
       args:
         chdir: /home/ubuntu/regional-rail-explorer
 
+    # - name: ensure caniuse-lite is up-to-date
+    #   shell: npx browserslist@latest --update-db
+    #   args:
+    #     chdir: /home/ubuntu/regional-rail-explorer
+
     - name: build application
       shell: npm run build
       args:
         chdir: /home/ubuntu/regional-rail-explorer
+      register: build_cmd
+      failed_when: build_cmd.rc != 0
```

Comment:
It's surprising to me that this isn't the default!

Reply:
This isn't right, 0 already means success https://docs.ansible.com/ansible/latest/collections/ansible/builtin/shell_module.html#return-rc

Reply:
whoopsie

Reply:
wait, but it's a `!=` @mathcolo...